polkadot-test-service/src/chain_spec.rs

```diff
@@ -0,0 +1,162 @@
+		get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+	]
+}
+
+/// Helper function to create polkadot GenesisConfig for testing
+fn polkadot_testnet_genesis(
+	initial_authorities: Vec<(AccountId, AccountId, BabeId, GrandpaId, ValidatorId)>,
+	root_key: AccountId,
+	endowed_accounts: Option<Vec<AccountId>>,
+) -> polkadot_test_runtime::GenesisConfig {
+	use polkadot_test_runtime as polkadot;
+
+	let endowed_accounts: Vec<AccountId> = endowed_accounts.unwrap_or_else(testnet_accounts);
+
+	const ENDOWMENT: u128 = 1_000_000 * DOTS;
+	const STASH: u128 = 100 * DOTS;
+
+	polkadot::GenesisConfig {
+		system: Some(polkadot::SystemConfig {
+			code: polkadot::WASM_BINARY.to_vec(),
+			changes_trie_config: Default::default(),
+		}),
+		indices: Some(polkadot::IndicesConfig { indices: vec![] }),
+		balances: Some(polkadot::BalancesConfig {
+			balances: endowed_accounts
+				.iter()
+				.map(|k| (k.clone(), ENDOWMENT))
+				.collect(),
+		}),
+		session: Some(polkadot::SessionConfig {
+			keys: initial_authorities
+				.iter()
+				.map(|x| {
+					(
+						x.0.clone(),
+						x.0.clone(),
+						polkadot_test_runtime::SessionKeys {
+							babe: x.2.clone(),
+							grandpa: x.3.clone(),
+							parachain_validator: x.4.clone(),
+						},
+					)
+				})
+				.collect::<Vec<_>>(),
+		}),
+		staking: Some(polkadot::StakingConfig {
+			minimum_validator_count: 1,
+			validator_count: 2,
+			stakers: initial_authorities
+				.iter()
+				.map(|x| {
+					(
+						x.0.clone(),
+						x.1.clone(),
+						STASH,
+						polkadot::StakerStatus::Validator,
+					)
+				})
+				.collect(),
+			invulnerables: initial_authorities.iter().map(|x| x.0.clone()).collect(),
+			force_era: Forcing::NotForcing,
+			slash_reward_fraction: Perbill::from_percent(10),
+			..Default::default()
+		}),
+		babe: Some(Default::default()),
+		grandpa: Some(Default::default()),
+		authority_discovery: Some(polkadot::AuthorityDiscoveryConfig { keys: vec![] }),
+		parachains: Some(polkadot::ParachainsConfig {
+			authorities: vec![],
+		}),
+		registrar: Some(polkadot::RegistrarConfig {
+			parachains: vec![],
+			_phdata: Default::default(),
+		}),
+		claims: Some(polkadot::ClaimsConfig {
+			claims: vec![],
+			vesting: vec![],
+		}),
+		vesting: Some(polkadot::VestingConfig { vesting: vec![] }),
+		sudo: Some(polkadot::SudoConfig { key: root_key }),
+	}
+}
+
+/// Can be called for a `Configuration` to check if it is a configuration for the `Test` network.
+pub trait IdentifyVariant {
+	/// Returns if this is a configuration for the `Test` network.
+	fn is_test(&self) -> bool;
+}
+
+impl IdentifyVariant for Box<dyn ChainSpec> {
+	fn is_test(&self) -> bool {
+		self.id().starts_with("test") || self.id().starts_with("tst") // TODO not sure why I need a 3 chars variant
```

Comment:
@bkchr Do I need this?

Reply:
IDK? You added this?

Reply:
No I mean...

I need the function `is_test` itself for starting the correct runtime when using `start_collator`

But I don't know if I need to test with "tst". All the other runtimes in this repo do that but I couldn't find a place where this test with 3 letters were useful. Well, I might just keep it for consistency with the others or I might just remove it because it's dead code